test/integration/tox.ini

```diff
@@ -0,0 +1,14 @@
+[tox]
```

Comment:
I see we had a different requirement on the Ansible version because of the issue with `delegate_to`, but we could probably still get away with a single tox.ini file in the repository. Earlier this year we've unified /tox.ini and /utils/tox.ini to avoid having to maintain two sets of configuration files.